docs/using-radicle/creating-projects.md

```diff
@@ -0,0 +1,73 @@
+---
+id: creating-projects
+title: Creating projects
+---
+
+Creating a project is as easy as initializing a repository in Git. In the
+Upstream client, you can create a project from scratch or from an existing
+repository. Creating a project on Radicle will publish it to the network, making
+it available to any of your connected peers.
+
+## Starting a new project
+
+If starting a new project from scratch, choose `Create from a new repository` in
+the **Start a new project** modal. This will prompt you to choose a location on
+your machine for the repository to be created. At the moment, [projects can only
+be created through the Upstream client][fa].
+
+![Start New Project][np]
+
+This will initalize a Git repository (with the name you've given it) in the
+location you choose and publish it to the Radicle network. Note, that when a
+project is published to the network, it will be able to be replicated by any
+connected peers. 
+
+## Publishing an existing project
+
+If you have a repository that you'd like to publish to the Radicle network,
+choose the 'Create from an exisiting repository' in the **Start a new project**
+modal. This will allow you to choose the repository and branch you wish to
+publish to the network.
```

Comment:
```suggestion
modal. This will allow you to choose the repository and its configured default branch.
All branches present when you add the repository will be published.
```